ontology/mp.md

```diff
@@ -43,6 +43,22 @@ browsers:
   - label: Monarch
     title: Monarch Phenotype Page
     url: http://monarchinitiative.org/phenotype/MP:0000001
+usages:
+  - user: http://www.informatics.jax.org/vocab/mp_ontology
+    description: MGI annotates phenotypes of mouse models using the MP
+    examples:
+      - url: http://www.informatics.jax.org/vocab/mp_ontology/MP:0008762
+        description: "Term browser page for embryonic lethality showing information about the term including definition, placement in the MP hierarchy, and link to mouse models annotated to this term or any of its decendants"
+  - user: https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MP:0000001
+    description: RGD annotates phenotypes associated with rat genes and alleles using the MP
+    examples:
+      - url: https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MP:0008762
+        description: "Term browser page for embryonic lethality showing information about the term including definition, placement in the MP hierarchy, and link to annotations to this term or any of its decendants"
+  - user: https://www.mousephenotype.org/
+    description: "IMPC annotates abnormal phenotypes of mice carrying null alleles found following the application of a standardised set of physiological tests"
+    examples:
+      - url: https://www.mousephenotype.org/data/genes/MGI:97490#phenotypesTab
```

Comment:
I don't see a use of MP on this page. @matentzn suggests something like https://www.mousephenotype.org/data/phenotypes/MP:0013522 instead.

Reply:
The significant phenotype table on the page uses MP terms for all the phenotype terms. They just don't show the MP IDs.
![IMPC_pheno](https://user-images.githubusercontent.com/10979793/133832732-bce8d0dc-08c3-451a-a8d6-97bd3fe6f0ea.png)

Reply:
I can switch to the other page if you think that is easier to understand or better as an example.

Reply:
Just because we are still clarifying the guidelines, I think it would be great if you would use URLs with MP ids in it.. :) Thank yOU!